### PR TITLE
Feature/release notes 103012 verzoek data naar pv

### PR DESCRIPTION
**Pull request by Ayub-ritense**

### CODE REVIEW

File: release-notes/major11/11.2.0/valtimo-backend-libraries.md [hunk 1]
```diff
@@ -74,6 +74,10 @@ The following features were added:
 
   Valtimo now uses a Spring managed ObjectMapper to map from JSON to classes and vice versa.
 
+* **Verzoak data mapping to a process variable**
+
+  `VerzoekPlugins` now provides the ability to map the verzoek data directly to a process variable by adding a `pv` prefix to the chosen `target` on a specified copy strategy.
```
Comment (Klaas-Ritense) [inline release-notes/major11/11.2.0/valtimo-backend-libraries.md:79]:
```suggestion
* **Verzoek data mapping to a process variable**

  The Verzoek plugin now provides the ability to map the verzoek data directly to a process variable by adding a `pv` prefix to the chosen `target` on a specified copy strategy.
```

**Review by Klaas-Ritense — APPROVED** [review]

**Comment by Ayub-ritense** [comment]
> Wrong target branch. Please add it to the release notes of 11.3.0

Shall i maken one: release/11.3.0

**Comment by Klaas-Ritense** [comment]
> Shall i maken one: release/11.3.0 ?

Please make one: `development/11.3.0`

File: release-notes/major11/11.3.0/valtimo-backend-libraries.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+## New Features
+
+The following features were added:
+
+* **Export case definitions**
+
+  Case definitions can now be exported as an archive, containing all process definitions, forms, form flows,
+  process links and the document definition that make up the case definition. See 
+  [this page](/using-valtimo/case/exporting-case-definitions.md#export) for more information.
```
Comment (Klaas-Ritense) [inline release-notes/major11/11.3.0/valtimo-backend-libraries.md:11]: These features aren't part of 11.3.0

File: release-notes/major11/11.3.0/valtimo-backend-libraries.md
Comment (Klaas-Ritense) [inline release-notes/major11/11.3.0/valtimo-backend-libraries.md:1]: These files have to be linked in the SUMMARY.md

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**